e2e_tests/src/utils.ts

```diff
@@ -62,32 +65,6 @@ const setupBrowserWithExtension = async (webAppUrl: string) => {
     };
 };
 
-// set testing wallet.
```

Comment:
bit confused why this was removed in this commit, but I reckon it was not needed anymore.

Reply:
Sorry for that.